apps/client/src/common/components/view-params-editor/ViewParamsEditor.tsx

```diff
@@ -117,18 +117,30 @@ export default function ViewParamsEditor({ viewOptions }: EditFormDrawerProps) {
   const onParamsFormSubmit = (formEvent: FormEvent<HTMLFormElement>) => {
     formEvent.preventDefault();
 
-    const newParamsObject = Object.fromEntries(new FormData(formEvent.currentTarget));
+    const newParamsObject: Record<string, string> = {};
+
+    new FormData(formEvent.currentTarget).forEach((value, key) => {
+      // handle multi-select values by concating new values
+      if (key in newParamsObject) {
+        newParamsObject[key] = newParamsObject[key].concat(`,${value}`);
+      } else {
+        newParamsObject[key] = String(value);
+      }
+    });
+
     const newSearchParams = getURLSearchParamsFromObj(newParamsObject, viewOptions);
     setSearchParams(newSearchParams);
+
+    onClose();
   };
 
   return (
-    <Drawer isOpen={isOpen} placement='right' onClose={handleClose} variant='ontime' size='lg'>
-      <DrawerOverlay />
-      <DrawerContent>
-        <DrawerHeader>
-          <DrawerCloseButton size='lg' />
+    <DrawerRoot open={isOpen} placement='end' onOpenChange={handleClose} size='lg' trapFocus={false}>
```

Comment:
suggestion: it doesnt look like we need the trapFocus

```suggestion
    <DrawerRoot open={isOpen} placement='end' onOpenChange={handleClose} size='lg'>
```

Reply:
Removed in https://github.com/cpvalente/ontime/pull/1438/commits/c496c3658d431fed64b92f1f647895535b21c7e9